### PR TITLE
Create main page

### PR DESCRIPTION
**Pull request by H-R-G-P**

### CODE REVIEW

File: src/Entity/Version.php [hunk 1]
```diff
@@ -50,26 +50,26 @@ public function setIdDance(?Dance $id_dance): self
...
+    public function getViews(): int
     {
         return $this->views;
     }
 
-    public function setViews(?int $views): self
+    public function subView(): self
     {
-        $this->views = $views;
+        $this->views = ++$this->views;
```
Comment (b1514) [inline src/Entity/Version.php:72]: $this->views++;

File: templates/homepage/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,78 @@
+{% extends 'base.html.twig' %}
+
+{% block title %}Танцы Беларусі{% endblock %}
```
Comment (b1514) [inline templates/homepage/index.html.twig:3]: Create translations files

Comment (H-R-G-P) [reply templates/homepage/index.html.twig:3]: I don't understand what to do.

Comment (H-R-G-P) [reply templates/homepage/index.html.twig:3]: translations into which languages?

File: assets/controllers/hello_controller.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+ *
+ * Any element with a data-controller="hello" attribute will cause
+ * this controller to be executed. The name "hello" comes from the filename:
+ * hello_controller.js -> "hello"
+ *
+ * Delete this file or adapt it for your use!
+ */
+export default class extends Controller {
+    connect() {
+        this.element.textContent = 'Hello Stimulus! Edit me in assets/controllers/hello_controller.js';
```
[the remark this answers is not in the document]
Comment (H-R-G-P) [reply assets/controllers/hello_controller.js:14]: It's auto-generated and I delete it.

File: templates/homepage/index.html.twig (hunk 2 above)
Comment (b1514) [reply templates/homepage/index.html.twig:3]: Do not use russian text in twig. Use translations

**Pull request merged**